controllers/v1/programs.js

```
@@ -7,6 +7,7 @@
 
 // Dependencies
 const programsHelper = require(MODULES_BASE_PATH + '/programs/helper')
+const programsQueries = require(DB_QUERY_BASE_PATH + '/programs')
```

ESLint

[error] 'programsQueries' is assigned a value but never used.

(no-unused-vars)